scripts/force-rebuild-status-go.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eof pipefail
+
+file=./vendor/status-lib/vendor/status-go/build/bin/libstatus.so
```

Comment:
This doesn't work on Windows:
```
find: ‘./vendor/status-lib/vendor/status-go/build/bin/libstatus.so’: No such file or directory
```

In windows, the file is called `libstatus.dll`

Reply:
Use the variable @richard-ramos suggested solved this problem
Let me know if windows is happy now